terraform/main.tf

```diff
@@ -104,13 +111,36 @@ data "google_project" "project" {
 data "google_client_config" "default" {}
 
 module "cross_device" {
```

Comment:
Once this PR is in working state, we should refactor the `cross-device` module to be a standalone terraservice instead of a submodule. This will facilitate the integration with the accelerated-platforms repo, and will also make developing the `cross-device` service easier because of the reduced number of dependencies.